### PR TITLE
Added basic logging config to bot

### PR DESCRIPTION
**Pull request by Shivansh-007**

### CODE REVIEW

File: bot/__init__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+from loguru import logger
+import os
+
+# set up logging
+
+log_dir = 'log'
+log_file = log_dir + os.sep + 'gurkbot.log'
```
Comment (vivax3794) [inline bot/__init__.py:8]: maybe this should live in constants.py? also using pathlib would make it nicer

Comment (Shivansh-007) [reply bot/__init__.py:8]: Never used Pathlib hence did with os, sure will put that in the constants.py

**Comment by Shivansh-007** [comment]
@vivax3794 your change has been mentioned. Is it ok now?

**Comment by vivax3794** [comment]
looks good, just missing those type hints now

**Comment by vivax3794** [comment]
actually I just tested this pr and
```py
    def should_rotate(message: loguru.Message, file):
AttributeError: module 'loguru' has no attribute 'Message'
```

**Comment by Shivansh-007** [comment]
Tested locally and ran flake8, works well.

File: bot/__init__.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    """When should the bot rotate : Once in 1 week or if the size is greater than 5 MB."""
+    filepath = os.path.abspath(file.name)
+    creation = os.path.getmtime(filepath)
+    now = message.record["time"].timestamp()
+    max_time = 7 * 24 * 60 * 60  # 1 week in seconds
+    if file.tell() + len(message) > 5 * (2 ** 20):  # if greater than size 5 MB
+        return True
+    if now - creation > max_time:
+        return True
+
```
Comment (vivax3794) [inline bot/__init__.py:22]: It would make sense to explicitly return False at the end here

Comment (Shivansh-007) [reply bot/__init__.py]: done

**Review by vivax3794 — APPROVED** [review]
Looks nice!

**Review by ReneganRonin — COMMENTED** [review]
No issues so far. Looks good.

**Review by ReneganRonin — APPROVED** [review]

**Pull request merged**